api/internal/utils/utils.go

```diff
@@ -161,3 +162,10 @@ func LabelContains(labels map[string]string, reqLabels map[string]struct{}) bool
 
 	return false
 }
+
+// ValidateLuaCode validates lua syntax for input code, return nil
+// if passwd, otherwise a non-nil error will be returned
```

Comment:
passwd? Is this a typo?

Reply:
> passwd? Is this a typo?

@starsz oh god, yep, it's a typo. Good catch. Will fix it soon. Thanks.